simtools/layout/layout_array.py

```diff
@@ -297,11 +299,35 @@ def _initialize_coordinate_systems(self, center_dict=None):
 
         """
 
+        center_dict = {} if center_dict is None else center_dict
+
         self._array_center = TelescopePosition()
         self._array_center.name = "array_center"
         self._array_center.set_coordinates("corsika", 0.0 * u.m, 0.0 * u.m, 0.0 * u.m)
+        self._set_array_center_mercator(center_dict)
+        self._set_array_center_utm(center_dict)
+        try:
+            self._array_center.set_altitude(u.Quantity(center_dict.get("center_alt", np.nan * u.m)))
+        except TypeError:
```

Comment:
Are you sure there's a case where you get TypeError? Everything here should be defined so I don't see how it would happen (and you didn't write a test for it).